tests/twig-loader.php

```diff
@@ -7,6 +7,7 @@
 
 $loader = new FilesystemLoader([], __DIR__);
 
+$loader->addPath(__DIR__ . '/EndToEnd'); // __main__
```

Comment:
Shouldn't the main path be `__DIR__`?

This loader can be used by other tests as well.